src/services/clinical/index.js

```diff
@@ -163,7 +186,10 @@ const getClinicalData = async (variables, Authorization) => {
 };
 
 const getClinicalErrors = async (programShortName, donorIds, Authorization) => {
-  const url = `${CLINICAL_SERVICE_ROOT}/clinical/program/${programShortName}/clinical-errors?donorIds=${donorIds}`;
+  const url = urlJoin(
+    `${CLINICAL_SERVICE_ROOT}/clinical/program/${programShortName}`,
```

Comment:
is it possible to separate the env variable from your string in the URL join arguments?

urlJoin(CLINICAL_SERVICE_ROOT, '/clinical...

Reply:
Updated